docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -25,6 +25,7 @@
 |spring.cloud.gateway.filter.secure-headers.xss-protection-header | 1 ; mode=block | 
 |spring.cloud.gateway.forwarded.enabled | true | Enables the ForwardedHeadersFilter.
 |spring.cloud.gateway.globalcors.cors-configurations |  | 
+|spring.cloud.gateway.globalcors.add-to-simple-url-handler-mapping | false | If global CORS config should be added to the URL handler.
```

Comment:
I'm pretty sure this file is auto generated. You need to add it to the json metadata file.

Reply:
Added to the metadata file.